### PR TITLE
kde-rounded-corners: init at unstable-2021-11-06

### PR DESCRIPTION
**Pull request by flexagoon**
###### Motivation for this change
Package [KDE-Rounded-Corners](https://github.com/matinlotfali/KDE-Rounded-Corners)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/kwin-decorations/kde-rounded-corners/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    rev = "d2eaac5a2ba7be9a0cd25b290c0a0e4635292ab0";
+    sha256 = "15zynbgyw010f9zz2b7qwhxjqf7lbisfgsqgwv1mxx851k2xn1di";
+  };
+
+  postConfigure = ''
+    substituteInPlace cmake_install.cmake \
+      --replace "${kdelibs4support}" "$out"
+  '';
+
+  nativeBuildInputs = [ cmake extra-cmake-modules wrapQtAppsHook kdelibs4support epoxy libXdmcp ];
```
Comment (SuperSandro2000) [inline pkgs/data/themes/kwin-decorations/kde-rounded-corners/default.nix:29]: Aren't the last 3 also libs which go to buildInputs?

**Comment by flexagoon** [comment]
@SuperSandro2000  please re-review

File: pkgs/data/themes/kwin-decorations/kde-rounded-corners/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, wrapQtAppsHook
+, kwin
+, kdelibs4support
+, epoxy
```
Comment (SuperSandro2000) [inline pkgs/data/themes/kwin-decorations/kde-rounded-corners/default.nix:8]:
```suggestion
, libepoxy
```

File: pkgs/data/themes/kwin-decorations/kde-rounded-corners/default.nix [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    sha256 = "0xbskf7jd03d2invfz1nnfc82klzvc784snw539n4kn6c6rc381p";
+  };
+
+  postConfigure = ''
+    substituteInPlace cmake_install.cmake \
+      --replace "${kdelibs4support}" "$out"
+  '';
+
+  nativeBuildInputs = [ cmake extra-cmake-modules wrapQtAppsHook ];
+  buildInputs = [ kwin kdelibs4support epoxy libXdmcp ];
```
Comment (SuperSandro2000) [inline pkgs/data/themes/kwin-decorations/kde-rounded-corners/default.nix:30]:
```suggestion
  buildInputs = [ kwin kdelibs4support libepoxy libXdmcp ];
```

**Pull request merged**